starter-code/pizza.js

```diff
@@ -17,69 +17,205 @@ var state = {
   greenPeppers: true,
   whiteSauce: false,
   glutenFreeCrust: false
-}
+};
 
 // This function takes care of rendering the pizza based on the state
 // This function is triggered once at the begining and everytime the state is changed
 function renderEverything() {
-  renderPepperonni()
-  renderMushrooms()
-  renderGreenPeppers()
-  renderWhiteSauce()
-  renderGlutenFreeCrust()
-
-  renderButtons()
-  renderPrice()
+  renderPepperonni();
+  renderMushrooms();
+  renderGreenPeppers();
+  renderWhiteSauce();
+  renderGlutenFreeCrust();
+
+  renderButtons();
+  renderPrice();
 }
 
 function renderPepperonni() {
-  document.querySelectorAll('.pep').forEach(function($pep){
+  document.querySelectorAll(".pep").forEach(function(pep) {
     if (state.pepperonni) {
-      $pep.style.visibility = "visible";
+      pep.style.visibility = "visible";
+    } else {
+      pep.style.visibility = "hidden";
     }
-    else {
-      $pep.style.visibility = "hidden";
-    }
-  })
+  });
 }
 
 function renderMushrooms() {
   // Iteration 1: set the visibility of `<section class="mushroom">`
+  document.querySelectorAll(".mushroom").forEach(function(mush) {
+    if (state.mushrooms) {
+      mush.style.visibility = "visible";
+    } else {
+      mush.style.visibility = "hidden";
+    }
+  });
 }
 
 function renderGreenPeppers() {
-  // Iteration 1: set the visibility of `<section class="green-pepper">`
+  document.querySelectorAll(".green-pepper").forEach(function(gPep) {
+    if (state.greenPeppers) {
+      gPep.style.visibility = "visible";
+    } else {
+      gPep.style.visibility = "hidden";
+    }
+  });
 }
 
 function renderWhiteSauce() {
-  // Iteration 2: add/remove the class "sauce-white" of `<section class="sauce">`
+  // Iteration 2: add/remove the class "sauce-white" of `<section class="sauce">
+  if (state.whiteSauce) {
+    document.querySelector(".sauce").classList.add("sauce-white");
+  } else {
+    document.querySelector(".sauce").classList.remove("sauce-white");
+  }
 }
 
 function renderGlutenFreeCrust() {
   // Iteration 2: add/remove the class "crust-gluten-free" of `<section class="crust">`
+  if (state.glutenFreeCrust) {
+    document.querySelector(".crust").classList.add("crust-gluten-free");
+  } else {
+    document.querySelector(".crust").classList.remove("crust-gluten-free");
+  }
 }
 
 function renderButtons() {
   // Iteration 3: add/remove the class "active" of each `<button class="btn">`
+  // document.querySelectorAll(".btn").forEach(function(btn) {
+  //   btn.classList.toggle("active");
+  // });
+
```

Comment:
In order to reduce your code, you could do a loop here looping over the keys of the state object and checking its state (true or false) if true, you want to add the class active, if false you want to remove the class active. 

This way you do not need to repeat the same code for every ingredient :)